scientific/spectroscopy/specpr.ksy

```diff
@@ -0,0 +1,254 @@
+meta:
+  id: specpr
+  file-extension: spec

```

Comment:
As far as I see, none of the files at ftpext.cr.usgs.gov has this extension. Moreover, looks like this format came from classic UNIX background, which frequently used no extensions at all. Are you sure about `.spec`?